closed/OpenJ9.gmk

```diff
@@ -210,7 +233,12 @@ define openj9_copy_tree_impl
   @$(TOUCH) $1/$(OPENJ9_MARKER_FILE)
 endef
 
-$(eval $(call generated_target_rules_build,build_jdk,$(BUILD_JDK)))
+# Temporary fix until we work out how to handle the build_jdk for cross compiles
```

Comment:
Does generating the rule cause problems if its not being run via the change to closed/make/Main.gmk?

Reply:
It certainly did at one point in time. I don't know if any of my more recent changes have made any difference to that. The generated_target_rules* behaviour is remarkably opaque, even after several hours of make --debug hacking.